test/test_custom_ops.py

```diff
@@ -2117,6 +2117,47 @@ def _(x, y):
         self.assertEqual(z, x + y)
         self.assertTrue(cpu_called)
 
+    def test_mutated(self):
+        @torch.library.custom_op(
+            "_torch_testing::numpy_sin_inplace", mutated_args={"x"}, device_types="cpu"
+        )
+        def numpy_sin_inplace(x: Tensor) -> None:
+            x_np = x.numpy()
+            np.sin(x_np, out=x_np)
+
+        x = torch.randn(3)
+        version = x._version
+        expected = x.sin()
+        numpy_sin_inplace(x)
+        self.assertEqual(x, expected)
+        self.assertGreater(x._version, version)
+
+        @torch.library.custom_op("_torch_testing::f", mutated_args="yzwv")
```

Comment:
I don't think this should be allowed! This is not going to work for any non-trivially named arguments and is very confusing to read.

Reply:
Sure